packages/lodestar/src/network/gossip/interface.ts

```diff
@@ -105,3 +106,11 @@ export type GossipMessageValidatorFn = (message: GossipObject, subnet?: number)
 export interface ILodestarGossipMessage extends Message {
   messageId: string;
 }
+
+export class GossipValidationError extends Error {
+  public code: string;
+  constructor(code: string, message?: string) {
```

Comment:
ain't this code suppose to be enum?

Reply:
I got error "Only numeric enums can have computed members" if I import error constants from gossipsub to make an enum in lodestar. I think gossipsub can make an enum instead if we want to.